packages/google_identity_services_web/README.md

```diff
@@ -24,18 +24,16 @@ There are two ways to load the JS SDK in your app.
 
 The most performant way is to modify your `web/index.html` file to insert a
 script tag [as recommended](https://developers.google.com/identity/gsi/web/guides/client-library).
-Place the `script` tag in the `<head>` of your site, next to the script tag that
```

Comment:
I wasn't 100% sure what to do here. Is this correct?

Reply:
I'm not sure because I haven't tried this new index, but leaving it in the `head` wouldn't hurt. This needs to be tested, however (I suspect google maps had a similar change)

Reply:
I actually get this warning in the Chrome dev console for google maps now. I tried using the script they give but it doesn't work.

```console
Google Maps JavaScript API has been loaded directly without loading=async. This can result in suboptimal performance. For best-practice loading patterns please see https://goo.gle/js-api-loading
```

Reply:
Ah I see https://github.com/flutter/flutter/issues/132150 tracking that